src/Language/PureScript/Interactive/Message.hs

```diff
@@ -37,18 +41,18 @@ prologueMessage = unlines
   , "Type :? for help"
   ]
 
+noInputMessage :: String
+noInputMessage = unlines
+  [ "purs repl: No input files; try running `pulp psci` instead."
+  , "For help getting started, visit " ++ guideURL
+  , "Usage: For basic information, try the `--help' option."
+  ]
+
 supportModuleMessage :: String
 supportModuleMessage = unlines
-  [ "PSCi requires the psci-support package to be installed."
-  , "You can install it using Bower as follows:"
-  , ""
-  , "  bower i purescript-psci-support --save-dev"
-  , ""
-  , "Or using psc-package:"
-  , ""
-  , "  psc-package install psci-support"
-  , ""
-  , "For help getting started, visit https://github.com/purescript/documentation/blob/master/PSCi.md"
+  [ "purs repl: psci-support is not loaded; try running `pulp psci` instead."
```

Comment:
Now that the "no input files" case is separately handled, I think the current message is actually more appropriate. I'm specifically thinking about a case where someone *has* invoked `purs repl` via `pulp psci` but `psci-support` is not installed.